### ISSUE TITLE
Elements must only use allowed ARIA attributes

### ISSUE THREAD

**Issue by will0684** [gcdevops/HRWhiteListing#95]
Elements must only use allowed ARIA attributes, the following is incorrectly implemented in the app:

<div class="o_searchview" role="search" aria-autocomplete="list">

The "aria-autocomplete" attribute should be removed from the div and instead added to the child <input>. aria-autocomplete should be used on input elements and not on the search landmark created by role="search".

You can read more about this here:

https://www.digitala11y.com/aria-autocomplete-properties/

**Comment by shawnthompson** [comment]
Can this be achieved with using basic HTML?
```
<form action="/action_page.php" autocomplete="on">
First name:<input type="text" name="fname"><br>
Last name: <input type="text" name="lname"><br>
E-mail: <input type="email" name="email" autocomplete="off"><br>
<input type="submit">
</form>
```
Please don't use `br` elements like that though!

**Comment by will0684** [comment]
The naming appears to be an issue in ARIA along with HTML5, you can read about it here:

https://github.com/w3c/aria-practices/issues/639

Basically the html5 input autocomplete attribute is different than the aria-autocomplete. You can see the details here:

aria-autocomplete:

https://www.digitala11y.com/aria-autocomplete-properties/

input autocomplete attribute:

https://www.w3schools.com/tags/att_input_autocomplete.asp
